src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/organisations/OrganisationsSynchronisationService.kt

```
@@ -429,36 +429,43 @@ class OrganisationsSynchronisationService(
-              nomisInternetAddress.toDpsCreateOrganisationWebAddressRequestForUpdateSwitch(event.corporateId),
-            ).also { dpsOrganisationWebAddress ->
-              telemetry["dpsOrganisationWebAddressId"] = dpsOrganisationWebAddress.organisationWebAddressId
+              val dpsOrganisationWebAddress = dpsApiService.createOrganisationWebAddress(
+                nomisInternetAddress.toDpsCreateOrganisationWebAddressRequestForUpdateSwitch(event.corporateId),
+              ).also { dpsOrganisationWebAddress ->
+                telemetry["dpsOrganisationWebAddressId"] = dpsOrganisationWebAddress.organisationWebAddressId
+              }
+              webAddressMappingCreator.tryToCreateMapping(
+                OrganisationsMappingDto(
+                  nomisId = event.internetAddressId,
+                  dpsId = "${dpsOrganisationWebAddress.organisationWebAddressId}",
+                  mappingType = OrganisationsMappingDto.MappingType.NOMIS_CREATED,
+                ),
+                telemetry,
+              )
+            } else {
+              dpsApiService.updateOrganisationEmail(
+                mapping.dpsId.toLong(),
+                nomisInternetAddress.toDpsUpdateOrganisationEmailRequest(),
+              )
             }
-            webAddressMappingCreator.tryToCreateMapping(
-              OrganisationsMappingDto(
-                nomisId = event.internetAddressId,
-                dpsId = "${dpsOrganisationWebAddress.organisationWebAddressId}",
-                mappingType = OrganisationsMappingDto.MappingType.NOMIS_CREATED,
-              ),
-              telemetry,
-            )
-          } else {
-            dpsApiService.updateOrganisationEmail(
-              mapping.dpsId.toLong(),
-              nomisInternetAddress.toDpsUpdateOrganisationEmailRequest(),
-            )
           }
         }
+      } ?: run {
+        telemetryClient.trackEvent(
```

previously this would silently fail and consume message